index.d.ts

```diff
@@ -254,9 +254,14 @@ declare namespace execa {
 		killed: boolean;
 
 		/**
-		The signal that was used to terminate the process.
+		The name of the signal that was used to terminate the process.
 		*/
 		signal?: string;
+
+		/**
+		A human-friendly description of the signal that was used to terminate the process.
```

Comment:
Can you mention that it's already included in the default Execa error message? So users don't try to be smart and include it themselves and getting it double.

Can you also include a short example of what the description looks like for a signal? (readme too)

Reply:
We should also be explicit about that there might be signals with a `signalDescription`. So users should assume that `signalDescription` will be defined if `signal` is.

Reply:
Fixed in a92abc0c1ed0e32868dcc5ed8b9106116001e838